Beforeget-iOS/Beforeget-iOS/Network/APIModel/MainModel/MainResponse.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MainResponse.swift
+//  Beforeget-iOS
+//
+//  Created by soyeon on 2022/01/18.
+//
+
+import Foundation
+
+struct MainResponse: Codable {
+    let status: Int
+    let success: Bool
+    let message: String
+    let data: Main
+}
```

Comment:
요런건 공통으로 받는 데이터이기 때문에 공통으로 사용할 수 있도록 구성하는 것도 좋을 것 같습니다.
```suggestion
struct Response<T: Decodable>: Decodable {
    let status: Int
    let success: Bool
    let message: String
    let data: T
}
```